### PR TITLE
Fix crash on empty `${}` variable expansion

### PR DESCRIPTION
**Pull request by rcari**
This fix got lost in #704 reworks.

@brentleyjones

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -8,27 +8,27 @@ jobs:
...
+      - name: Build
+        run: swift build
+      - name: Test
+        run: swift test 2>&1 | xcpretty
+      - name: Gen fixtures
+        run: scripts/gen-fixtures.sh
+      - name: Check fixtures
+        run: scripts/diff-fixtures.sh
+      - name: Build fixtures
+        run: scripts/build-fixtures.sh
```
Comment (brentleyjones) [inline .github/workflows/ci.yml]: Can you make this as a separate PR that we can merge on its own?

Comment (rcari) [reply .github/workflows/ci.yml]: https://github.com/yonaskolb/XcodeGen/pull/707

Comment (brentleyjones) [reply .github/workflows/ci.yml]: Able to rebased now.

Comment (rcari) [reply .github/workflows/ci.yml]: Done!

**Comment by brentleyjones** [comment]
Thank you @rcari 😄.
